### PR TITLE
📱(frontend) fix mobile dashboard layout

### PR DESCRIPTION
**Pull request by rlecellier**
Our mobile version of the dashboard needed some improvments
See https://github.com/openfun/richie/issues/2097

This PR fix: 
* Sidebar right margin
* Sidebar navigation select 

image

### CODE REVIEW

File: src/frontend/scss/objects/_course_glimpses.scss [hunk 1]
```diff
@@ -75,6 +75,9 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
   @include media-breakpoint-up(lg) {
     @include sv-flex(1, 0, calc(25% - #{$r-course-glimpse-gutter * 2}));
   }
+  @include media-breakpoint-down(lg) {
+    max-width: 100%;
+  }
```
Comment (jbpenrath) [inline src/frontend/scss/objects/_course_glimpses.scss:80]: This change impacts all course glimpse all over the CMS. Did you check the impact of this change on Search page, home page, organization detail page, etc. You can easily compare the difference between our local version and the online demo version (https://demo.richie.education)

File: src/frontend/js/pages/TeacherDashboardCoursesLoader/index.spec.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { render, screen } from '@testing-library/react';
 import fetchMock from 'fetch-mock';
 import { IntlProvider } from 'react-intl';
 
+import { CunninghamProvider } from '@openfun/cunningham-react';
```
Comment (jbpenrath) [inline src/frontend/js/pages/TeacherDashboardCoursesLoader/index.spec.tsx:7]: This had not been done in #2106 ?

File: src/frontend/js/widgets/Dashboard/components/DashboardSidebar/components/NavigationSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    id: 'components.DashboardSidebar.settingsLinkLabel',
+    description: 'label of the settings link',
+    defaultMessage: 'Settings',
+  },
+  responsiveNavLabel: {
+    id: 'components.DashboardSidebar.responsiveNavLabel',
+    description: 'a11y related label for select input used to navigate on responsive',
+    defaultMessage: 'Navigate to',
+  },
+});
```
Comment (jbpenrath) [inline src/frontend/js/widgets/Dashboard/components/DashboardSidebar/components/NavigationSelect.tsx:17]: Copy/pasta failed. Messages id are wrong. 

`DashboardSidebar` -> `NavigationSelect`

File: src/frontend/js/widgets/Dashboard/components/TeacherDashboardCourseSidebar/index.spec.tsx [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { MemoryRouter, Route, Routes } from 'react-router-dom';
 import { render, screen } from '@testing-library/react';
 import { IntlProvider, createIntl } from 'react-intl';
 import { QueryClientProvider } from '@tanstack/react-query';
+import { CunninghamProvider } from '@openfun/cunningham-react';
```
Comment (jbpenrath) [inline src/frontend/js/widgets/Dashboard/components/TeacherDashboardCourseSidebar/index.spec.tsx:6]: Ditto

File: src/frontend/js/pages/TeacherDashboardCoursesLoader/index.spec.tsx (hunk 2 above)
Comment (rlecellier) [reply src/frontend/js/pages/TeacherDashboardCoursesLoader/index.spec.tsx:7]: There what's happen on the course page, look's ok to me:
image

File: src/frontend/scss/objects/_course_glimpses.scss (hunk 1 above)
Comment (rlecellier) [reply src/frontend/scss/objects/_course_glimpses.scss:80]: There what's happen on the course page, look's ok to me:
image

File: src/frontend/js/pages/TeacherDashboardCoursesLoader/index.spec.tsx (hunk 2 above)
Comment (rlecellier) [reply src/frontend/js/pages/TeacherDashboardCoursesLoader/index.spec.tsx:7]: in #2106 we bump cunningham to the last version
Here we've add a cunningham select in our sidebar, so cunningham provider is now needed.

File: src/frontend/scss/objects/_course_glimpses.scss (hunk 1 above)
Comment (rlecellier) [reply src/frontend/scss/objects/_course_glimpses.scss:80]: i didn't find a blog post with glimse on our demo site but all other pages are ok.

**Review by jbpenrath — COMMENTED** [review]
LGTM ✅

**Review by jbpenrath — APPROVED** [review]

**Pull request merged**